src/lib/zod/index.ts

```diff
@@ -8,12 +8,18 @@ import {
   zStringISODateUTC,
   zBytes32,
   zEthAddress,
-} from '../../../src/forms/formHelpers';
+  zUsername,
+  zCircleName,
+} from '../../forms/formHelpers';
 
 const PERSONAL_SIGN_REGEX = /0x[0-9a-f]{130}/;
 
 export const sha256HashString = z.string().length(64);
 
+export const circleLandingInput = z.object({
```

Comment:
i think it's an anti-pattern to put all of these schemas into one file, instead of co-locating them with the code that uses them. and in this case it appears that `circleLandingInput` isn't used anywhere, but the linter doesn't catch this because it's exported

Reply:
Good catch on that one. For circle and username I was frustrated that backend and front end had different validation and duplicated opinions about how to validate which is why I put them here.